crates/net/downloaders/src/headers/metrics.rs

```diff
@@ -0,0 +1,40 @@
+use metrics::Counter;
+use reth_interfaces::p2p::error::DownloadError;
+use reth_metrics_derive::Metrics;
+
+/// The header downloader metrics.
+#[derive(Metrics)]
+#[metrics(scope = "downloaders_headers")]
```

Comment:
metrics will translate to `downloaders_headers` and recommends using `.` as a separator

```suggestion
#[metrics(scope = "downloaders.headers")]
```

Reply:
yeah, we need to revert that PR that prohibits usage of dots first

Reply:
opened https://github.com/paradigmxyz/reth/pull/1018